open-banking-brasil-financial-api-1_ID3.md

```diff
@@ -319,6 +319,7 @@ In addition, the confidential client
 5. shall support refresh tokens
 6. shall not populate the `acr` claim with required values
 7. shall require the `acr` claim as an essential claim
+8. shall support all authentication methods specified in clause 5.2.2-14 of [Financial-grade API Security Profile 1.0 - Part 2: Advanced][FAPI-1-Advanced] including diferent combinations of the methods to send requests (using [PAR] or not - item 11) and response modes (using [JARM] or not - item 2).
```

Comment:
By this i understand that you want to require TPPs to support JARM? Note that JARM is going to be removed from FAPI 2.0 and has low client adoption in the library community. This is why JARM support was allowed for Banks to offer it but did not make it mandatory for TPPs to support it. This is a very big change for TPPs.

Reply:
Just to be clear - i originally wrote it this way because of significant concerns regarding interoperability for TPP client libraries.